src/main/java/org/usfirst/frc1745/deepspace2019/Controls.java

```diff
@@ -44,11 +57,23 @@ public double getRightY(double deadzone) {
         return y;
     }
 
-    public double getRightX(double deadzone) {
-        double x = joystick.getRawAxis(RIGHT_X);
-        if(Math.abs(x) < deadzone) {
-            return 0.0;
-        } 
-        return x;
+    public boolean getRightBumper() {
+        return joystick.getRawButton(RIGHT_BUMPER_ID);
+    }
+
+    public JoystickButton getRightJoystickBumper(){
+        return new JoystickButton(joystick, RIGHT_BUMPER_ID);
+    }
+
+    public JoystickButton getRightTrigger() {
+        return new JoystickButton(joystick, RIGHT_TRIGGER_ID);
```

Comment:
Let's go ahead and get this to work like we discussed yesterday. To recap, if the raw axis value is greater than x deadband (for example, `.1`) return true, otherwise return false. Let's go ahead and make a `getLeftTrigger()` method while we're at it.